forge-gui/res/cardsfolder/upcoming/sunken_palace.txt

```diff
@@ -0,0 +1,9 @@
+Name:Sunken Palace
+ManaCost:no cost
+Types:Land Cave
+K:CARDNAME enters the battlefield tapped.
+A:AB$ Mana | Cost$ T | Produced$ U | SpellDescription$ Add {U}.
+A:AB$ Mana | Cost$ 1 U T ExileFromGrave<5/Card>| Produced$ U | TriggersWhenSpent$ TrigCopy | SpellDescription$ Add {U}. When you spend this mana to cast a spell or activate an ability, copy that spell or ability. You may choose new targets for the copy. (Mana abilities can't be copied.)
+SVar:TrigCopy:Mode$ SpellAbilityCast | ValidSA$ Spell,Activated | ValidActivatingPlayer$ You | Execute$ TrigCopyMain | TriggerDescription$ When you spend this mana to cast a spell or activate an ability, copy that spell or ability. You may choose new targets for the copy. (Mana abilities can’t be copied.)
```

Comment:
This is not working

Reply:
?

Reply:
You're right it did work. It's just hard to use the "special" mana in testing